pkg/recommender.py

```diff
@@ -441,21 +441,44 @@ def _schedule_evt(self):
     Send the morning message at 10 am
     '''
     next_morning = ''
+    schedule_evts = [(10,'999'),(23,'998')] #(hour, event_id)
+    #early to late
+    evt_count = 0
+
     while True:
+      idx = evt_count%len(schedule_evts)
+      hour, event_id = schedule_evts[idx]
+
       now = datetime.now()
 
       #check if we have passed 10 am yet
-      if now.hour >= 10: #then you are currently in the day before
+      if idx == 0 and now.hour >= hour: #then you are currently in the day before
+        next_morning = now.replace(hour=schedule_evts[1][0], minute=0, second=0, microsecond=0)
+        #evening must be sent
+        message_id = schedule_evts[1][1]
+      if idx == 0 and now.hour < hour:
+        next_morning = now.replace(hour=schedule_evts[0][0], minute = 0, second = 0, microsecond =0)
+        #morning message must be sent
+        message_id = schedule_evts[0][1]
+      elif idx == 1 and now.hour < hour:
+        next_morning = now.replace(hour = schedule_evts[1][0], minute = 0, second = 0, microsecond = 0)
+        #evening message must be sent
+        message_id = schedule_evts[1][1]
```

Comment:
wow, man, this really take me a while to understand what you are doing.
I want to make it simple for you. That's why my pseudo code suggests we always start from morning. 

I see you want to do sth more advanced, identify if next event should be morning or evening.

Let me craft a simple solution for you. This is not flexible, which means it can only handle two events. Later when they give a requirement of another event, this will fail